src/lines-yardstick.coffee

```diff
@@ -32,32 +33,27 @@ class LinesYardstick
     lineOffset = lineNode.getBoundingClientRect().left
     targetLeft += lineOffset
 
-    textNodeStartColumn = 0
-    for textNode in textNodes
-      {length: textNodeLength, textContent: textNodeContent} = textNode
-      textNodeRight = @clientRectForRange(textNode, 0, textNodeLength).right
+    textNodeComparator = (textNode, position) =>
```

Comment:
How would you feel about inlining these comparators? Seems like it increasing locality increases clarity as well in this case.